nextflow.config

```diff
@@ -317,7 +324,7 @@ profiles {
     test_malt               { includeConfig 'conf/test_malt.config' }
     test_falco              { includeConfig 'conf/test_falco.config' }
     test_fastp              { includeConfig 'conf/test_fastp.config' }
-    test_adapterremoval     { includeConfig 'conf/test_adapterremoval.config' }
+    test_alternativepreprocessing        { includeConfig 'conf/test_alternativepreprocessing.config' }
     test_bbduk              { includeConfig 'conf/test_bbduk.config' }
     test_prinseqplusplus    { includeConfig 'conf/test_prinseqplusplus.config' }
```

Comment:
May want to adjust the formatting so the starting braces are all aligned in this block of the config.

Reply:
HarshilAlignment™️! [^1]

[^1]: https://nf-co.re/docs/contributing/code_editors_and_styling/harshil_alignment